s3/src/test/scala/akka/stream/alpakka/s3/impl/S3StreamSpec.scala

```diff
@@ -160,7 +159,8 @@ class S3StreamSpec(_system: ActorSystem)
     def bucketStatusPreparation(response: HttpResponse): Future[BucketAccess] = {
       val testedMethod = PrivateMethod[Future[BucketAccess]]('processCheckIfExistsResponse)
 
-      val result: Future[BucketAccess] = S3Stream invokePrivate testedMethod(response, materializer)
+      val result: Future[BucketAccess] = S3Stream invokePrivate testedMethod(response,
```

Comment:
TIL of ScalaTest's private method reflection API

Reply:
Should we use `.`-syntax instead of infix?

Reply:
I don't really have a preference here.

Reply:
this is a bit obscure anyway, so either way works for me. Probably a *little* nicer to use `.`-syntax.